<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#pushButton_connect {
	background-color: #008;
	color: #fff;
	border-radius:10px;
	width:130px;
	height:30px;
	border: 3px sold #fff;
}

#pushButton_2 {
	background-color: #008;
	color: #fff;
	border-radius:10px;
	width:130px;
	height:30px;
	border: 3px sold #fff;
}

#pushButton_3 {
	background-color: #008;
	color: #fff;
	border-radius:10px;
	width:130px;
	height:30px;
	border: 3px sold #fff;
}

#comboBox_2, #comboBox_baud, #comboBox_port {
	background-color: #005;
	color: #fff;
	width:130px;
	height:30px;
	border: 3px sold #fff;
}

#spinBox {
	background-color: #005;
	color: #fff;
	width:130px;
	height:30px;
	border: 3px sold #fff;
}

#Widget {
	background: #dcdcdc;
}



</string>
  </property>
  <widget class="QPushButton" name="pushButton_connect">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Conectar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_port">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Modo de Operação</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ativar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Manual</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Automático</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Desconectado</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>42</width>
     <height>29</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="specialValueText">
    <string notr="true"/>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="suffix">
    <string notr="true"/>
   </property>
   <property name="prefix">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Regar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_baud">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Limiar de Humidade</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
